### PR TITLE
makefile: remove str exists from sed

### PR DESCRIPTION
**Pull request by dougsland**
Resolves: https://github.com/containers/qm/issues/84

### CODE REVIEW

**Comment by dougsland** [comment]
@rhatdan I believe this is the fix, let's see the CI result.

File: Makefile [hunk 1]
```diff
@@ -6,7 +6,7 @@ DATADIR ?= $(PREFIX)/share
 LIBDIR ?= $(PREFIX)/lib
 SYSCONFDIR?=/etc
 QMDIR=/usr/lib/qm
-USER_NAMESPACE="user_namespace exists"
```
Comment (rhatdan) [inline Makefile:9]: This is wrong, we want to USER_NAMESPACE to not be removed which means the sed will not remove it.

Comment (rhatdan) [reply Makefile:9]: This change would cause it to always be removed.

**Review by Yarboa — APPROVED** [review]
LGTM

File: Makefile (hunk 1 above)
Comment (dougsland) [reply Makefile:9]: @rhatdan hum, that's why the spec always worked. The spec had this sed. Also, I cannot see `user_namespace exists` for Centos9 or Fedora (in the generated qm.if). Any other thing to me to dig?

Comment (dougsland) [reply Makefile:9]: @lsm5  I don't see user_namespace exists :(

# cat /etc/redhat-release
CentOS Stream release 9

# make qm.pp

# grep -rni user_namespace qm.if
135:	allow $1_t self:user_namespace all_user_namespace_perms;
437:	allow $1_container_domain self:user_namespace create;

# rpm -qa | grep -i selinux
libselinux-3.5-1.el9.x86_64
python3-libselinux-3.5-1.el9.x86_64
libselinux-utils-3.5-1.el9.x86_64
selinux-policy-38.1.14-1.el9.noarch
selinux-policy-targeted-38.1.14-1.el9.noarch
container-selinux-2.218.0-1.el9.noarch
rpm-plugin-selinux-4.16.1.3-23.el9.x86_64
selinux-policy-devel-38.1.14-1.el9.noarch

Comment (rhatdan) [reply Makefile:9]: Right, which means the sed statement will remove nothing.  The problem is sed will not work with "" So we needed to state something.  The idea is to only sed user_namespace if the class Does not exists

**Comment by rhatdan** [comment]
The question is what didn't this remove user_namespace on centos systems?

**Pull request closed without merging**